server/service/osquery_utils/queries.go

```diff
@@ -408,6 +383,55 @@ FROM logical_drives WHERE file_system = 'NTFS' LIMIT 1;`,
 	},
 }
 
+func ingestNetworkInterface(ctx context.Context, logger log.Logger, host *fleet.Host, rows []map[string]string) error {
+	if len(rows) == 0 {
+		logger.Log("component", "service", "method", "IngestFunc", "err",
+			"detail_query_network_interface expected 1 or more results")
+		return nil
+	}
+
+	host.PrimaryIP = rows[0]["address"]
+	host.PrimaryMac = rows[0]["mac"]
+	host.PublicIP = publicip.FromContext(ctx)
+	return nil
+}
+
+func directIngestDiskSpace(ctx context.Context, logger log.Logger, host *fleet.Host, ds fleet.Datastore, rows []map[string]string, failed bool) error {
```

Comment:
Nit: Did you mean to move the disk space query?  I think the file overall has some grouping of ingest vs. direct ingest functions, but it is pretty loosely enforced AFAICT. It's not really important for this PR so feel free to ignore my comment and merge :)

Reply:
The reason I moved this is so that it is defined immediately after `hostDetailQueries` where it is used. It seem to make more sense to keep the ingest functions close to where they are referenced.

Reply:
Ah, ok. I was thinking that all direct ingest functions were only used with the extraDetailQueries, but I now see that we use those with the hostDetailQueries too. Sorry for the distraction!